### DISCUSSION TITLE
Kirim Pesan ke Group

### DISCUSSION THREAD

**Discussion by rioastamal** [kirimwa-id/forum#4] [Roadmap]
Fitur untuk mengirim pesan ke Group 👥 

Vote up dan beri tanda 👍  &nbsp; jika fitur ini ingin segera diimplementasikan.

**Comment by ulilamri419** [comment]
Salam Mas Rio,

Fitur ini sangat cocok sekali buat planning design alerting saya.. mohon sekiranya bisa di implementasikan.. 

Terima kasih banyak.. 
sukses selalu
BR
Amri

**Reply by nasruloh96** [reply]
Setuju mas

**Reply by rioastamal** [reply]
@ulilamri419 @nasruloh96 fitur sudah dirilis check pengumuman di #19 😎

**Comment by rioastamal** [comment]
Fitur telah diimplementasikan. Lihat pengumuman di #19.
